packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -7,22 +7,38 @@ import { useBooleanState } from 'fxa-react/lib/hooks';
 import UnitRow from '../UnitRow';
 import UnitRowWithAvatar from '../UnitRowWithAvatar';
 import Modal from '../Modal';
+import AlertBar from '../AlertBar';
 import { AccountData } from '../AccountDataHOC/gql';
 
 export const Settings = ({ account }: { account: AccountData }) => {
   const [modalRevealed, revealModal, hideModal] = useBooleanState();
   const onSecondaryEmailConfirm = useCallback(() => {
     console.log('confirmed - resend verification code');
     hideModal();
+    revealAlertBar();
   }, [hideModal]);
```

Comment:
ESLint is complaining about this on build -- I believe `revealAlertBar` should be added to this array.